pkgs/development/python-modules/hatchling/default.nix

```diff
@@ -15,21 +15,21 @@
 , build
 , python
 , requests
-, toml
+, tomli
 , virtualenv
 }:
 
 let
   pname = "hatchling";
-  version = "0.25.0";
+  version = "1.0.0";
 in
 buildPythonPackage {
   inherit pname version;
   format = "pyproject";
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-k/bjZvGaOjZshVr6w3Jb7XaC1dAOlIaraFQKCth2ZII=";
+    hash = "d235a5fa8aff89e8d9d6d4033594aa4c3bc00ec5e31d3e80c153bfcf951b4f98";
```

Comment:
```suggestion
    sha256 = "d235a5fa8aff89e8d9d6d4033594aa4c3bc00ec5e31d3e80c153bfcf951b4f98";
```

Reply:
Ideally we would use hash instead of sha256, but what you've given it is no SRI hash.

Reply:
```patch
diff --git a/pkgs/development/python-modules/hatchling/default.nix b/pkgs/development/python-modules/hatchling/default.nix
index 6385b32a48c..ece07781e6b 100644
--- a/pkgs/development/python-modules/hatchling/default.nix
+++ b/pkgs/development/python-modules/hatchling/default.nix
@@ -21,7 +21,7 @@
 
 let
   pname = "hatchling";
-  version = "0.25.0";
+  version = "1.1.0";
 in
 buildPythonPackage {
   inherit pname version;
@@ -29,7 +29,7 @@ buildPythonPackage {
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-k/bjZvGaOjZshVr6w3Jb7XaC1dAOlIaraFQKCth2ZII=";
+    hash = "sha256-rKZ8K4RnVELBWTbvTWQ91USDfYPnU8O9uGEXfaFmkbk=";
   };
 
   # listed in backend/src/hatchling/ouroboros.py
```

Reply:
> Ideally we would use hash instead of sha256

I actually disagree in the case of `fetchPypi`. It's nice to be able to compare to the hash on pypi.org.